mylar/PostProcessor.py

```diff
@@ -3425,6 +3425,10 @@ def sendnotify(self, series, issueyear, issuenumOG, annchk, module, imageFile):
                 slack = notifiers.SLACK()
                 slack.notify("Download and Postprocessing completed", prline2, module=module)
 
+            if mylar.CONFIG.MATTERMOST_ENABLED:
+                mattermost = notifiers.MATTERMOST()
+                mattermost.notify("New comic available", prline2, module=module)
```

Comment:
We probably want to keep the notification standardized to what the other notifications read as.

Reply:
Also, is there a reason we're not including the image here? Does Mattermost not support images in messages?

Reply:
> Also, is there a reason we're not including the image here? Does Mattermost not support images in messages?

Not in the message body, as far as I could glean from their webhook documentation. I am currently working out how to properly include it into the attachment.

Reply:
> We probably want to keep the notification standardized to what the other notifications read as.

Makes sense. Adjusted in 589619eec388b19f2bb501f7ee243b33aaae8a23